source/locale/en/symbols.dic

```diff
@@ -7,10 +7,13 @@ complexSymbols:
 # identifier	regexp
 # Sentence endings.
 . sentence ending	(?<=[^\s.])\.(?=[\"'”’)\s]|$)
+。 sentence ending	(?<=[^\s.])\.(?=[\"'”’)\s]|$)
```

Comment:
Does the concept of sentence ending period really exist for ideographic period, i.e. is ideographic period used elsewhere than to end a sentence in the languages using it? And if yes, does this regexp really match its usage in this types of languages, i.e. should it be followed by a space and are the simple/double quotes used as in latin writing? I doubt it.

In any case, even if all of these assumptions were true, the regexp does not contain the ideographic period, so it is not correct.

If there is no concept of sentence ending period in languages using ideographs, you should remove the regexp instead and set the 'preserve' parameter to 'always' to avoid pitch issues of the synth in the sentence prosody.

At last, the same regexp is used two times, for normal period and ideographic period. I do not know which rule will be used by NVDA in this case; anyway, it does not make sense.

Reply:
Let's decide what to do after users test it.

Reply:
I do not know what user can test if ideographic period and English period are mapped to the same regexp. Please clarify it because it makes no sense to me.

Reply:
I think the ideographic period should be treated the same as the Latin period. Because the ideographic period is only used in certain languages. However, the Latin period is also used in these languages.

Reply:
> I think the ideographic period should be treated the same as the Latin period. Because the ideographic period is only used in certain languages. However, the Latin period is also used in these languages.

This is not the point. But it seems we do not understand each other.
What I am saying is that this does not make sense to use the same regexp for two complex symbols in the section `complexSymbols:`.
You have associated the same regexp for latin and ideographic sentence ending period.
When the text will be parsed and if the corresponding regexp is recognized, NVDA will report it either as latin sentence ending period or as ideographic sentence ending period, but it will choose one way to report. And the other way to report will never be used.

To be more concrete, try to associate a dummy pronunciation to latin sendence ending period and another one to ideographic sentence ending period. Then, try to generate a text to have each case reported. You will never be able to create such text since both regexps are the same.

I hope to have clarified my point now.

Reply:
I invited you as a collaborator. Please help me.